app/views/shared/_standards_edit.html.haml

```diff
@@ -0,0 +1,205 @@
+-# Expects locals: object
+- if object && (policy(object).admin_or_material_admin? || policy(object).author?)
+
+  = field_set_tag 'Standards' do
+    %div
+      Applied Standards
+      %div{:id => "applied_standards_div"}
+      %br
+    %div
+      %table{:class => "asn_add_table"}
+        %tr
+          %td
+            Add Standards
+          %td{:class => "asn_add_table_td_right"}
+            %input{:id => "add_statement_button", :type => "button", :value => "Add", :onclick => "window.toggleAddDiv()"}
+    %div{:id => "add_statement_div"}
+      %form
+        %table{:class => "asn_search_table"}
+          %tr
+            %td
+              Standard Document
+            %td
+              %select{:id => "asn_document_id", :multiple => false}
+                - StandardDocument.all.each do |s|
+                  %option{value: s.uri} #{s.name}
+            %td
+          %tr
+            %td
+              Notation
+            %td
+              %input{:id => "asn_statement_notation_query", :type => "text" }
+            %td
+          %tr
+            %td
+              Label
+            %td
+              %input{:id => "asn_statement_label_query", :type => "text" }
+            %td
+          %tr
+            %td
+              Description
+            %td
+              %input{:id => "asn_description_query", :type => "text" }
+            %td{:class => "asn_search_table_td_right"}
+              %input{:type => "button", :value => "Search",   :onclick => "window.searchASN()"}
+      %div{:id => "asn_search_results_message" }
+      %div{:id => "asn_search_results", :class => "asn_results_div"}
+      %br
+
+:javascript
+  jQuery(document).ready(function() {
+
+    jQuery("#asn_search_results_message").hide();
+    jQuery("#add_statement_div").hide();
+
+    //
+    // Form parameters sent to ASN as query params (through controller)
+    //
+    var PARAM_INPUTS = [    "asn_document_id",
+                            "asn_statement_notation_query",
+                            "asn_statement_label_query",
+                            "asn_description_query" ];
+
+
+    //
+    // Handle user pressing enter on form fields. Otherwise the
+    // modal popup is dismissed.
+    //
+    window.onKeyPress = function(event) {
+      if (event.which === 13) {
+        window.searchASN();
+        event.preventDefault();
+      }
+    };  // end window.onKeyPress()
+
+
+    //
+    // Need to attach these handlers here. HAML can't seem to correctly
+    // register them with :onkeypress maybe because this takes an 
+    // event parameter.
+    //
+    for(let p of PARAM_INPUTS) {
```

Comment:
Using `let` here might break older browsers - I'm not sure which browser versions we support now, but it would be good to check.